core/src/Http/HttpClientFactory.cs

```diff
@@ -37,12 +37,11 @@ internal interface IHttpClientFactory
 
     internal class HttpClientFactory : IHttpClientFactory
     {
-        // as per guidelines HttpClient should be a singleton instance in an application.
-        private static HttpClient _client;
-        private static readonly object LockObj = new object();
+        // The HttpClient is a singleton per ClientApplication so that we don't have a process wide singleton.
```

Comment:
> The HttpClient is a singleton per ClientApplication [](start = 10, length = 52)

I'm surprised that no prod code needed an update

Reply:
discussed offline. 

---
In reply to: [226472465](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1334#discussion_r226472465) [](ancestors = 226472465)